nemo_skills/code_execution/sandbox.py

```diff
@@ -253,6 +254,27 @@ async def is_proof_correct(self, pred_output, timeout=30.0):
             return "timeout"
         return determine_proof_status(output)
 
+    def _check_ready(self, timeout: float = 5.0) -> bool:
+        """Readiness check against the sandbox health endpoint."""
+        url = f"http://{self.host}:{self.port}/health"
+
+        try:
+            if self.ssh_server and self.ssh_key_path:
+                import sshtunnel_requests
+
+                sshtunnel_request = sshtunnel_requests.from_url(f"ssh://{self.ssh_server}:22", self.ssh_key_path)
+                response = sshtunnel_request.get(url=url, timeout=timeout)
+            else:
+                with httpx.Client() as client:
+                    response = client.get(url=url, timeout=timeout)
+            return response.status_code == 200
+        except Exception:
```

Comment:
actually on a second thought, I think we shouldn't have a blank exception here. If there is some error in this part, this can lead to the job just being stuck forever and could be hard to see why it happens. The exception shouldn't normally be thrown, right? So we can probably just remove that part and then should be good to merge